components/dashboard/src/teams/onboarding/OrgMemberAvatarInput.tsx

```diff
@@ -26,7 +26,7 @@ export const OrgMemberAvatarInput = ({ settings, setFeaturedMemberId }: Props) =
     const handleSelectionChange = useCallback(
         (selectedId: string) => {
             const member = members?.find((m) => m.userId === selectedId);
-            setFeaturedMemberId(selectedId || undefined);
+            setFeaturedMemberId(selectedId);
```

Comment:
This change was necessary to force the default value and not just make an empty update.